### DISCUSSION TITLE
Welcome to PTN-Ninja Discussions!

### DISCUSSION THREAD

**Discussion by gruppler** [gruppler/PTN-Ninja#50] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by findingfocus** [comment]
Hey, gruppler! I myself programmed Tak in Love2d and Lua, and once I finished it I looked up if anyone else had tried, and sure enough I was absolutely blown away at how fantastic your implementation was. I am a new developer, learning gamedev and web development, and I was wondering if you had any interest in coming on my twitch/youtube stream to talk about this project! No hard feelings if that doesn't sound fun to you, I just want to let you know that you did a great job and I'm very impressed

**Reply by gruppler** [reply]
Hello, thanks for the kind words and invitation. I'd be happy to chat about the project.

**Comment by findingfocus** [comment]
Wonderful! I couldn't find contact info for you, but if you shoot me an email paul@findingfocus.dev we can plan something out. Thank you for your time.
